linera-service/src/client.rs

```diff
@@ -1020,7 +1020,7 @@ pub async fn resolve_binary(name: &'static str, package: Option<&'static str>) -
 }
 
 #[cfg(not(debug_assertions))]
-pub async fn resolve_binary(name: &'static str) -> Result<PathBuf> {
+pub async fn resolve_binary(name: &'static str, _package: Option<&'static str>) -> Result<PathBuf> {
```

Comment:
It's only used in `client.rs` and in some tests. 
All the call sites are using the 2 arguments.
(but I don't think it runs in release mode, independently of this commit)

Reply:
Ah, I didn't know that! Thanks.

Reply:
But there _are_ more call sites to this, aren't there? In `linera-service`? Does `linera-service` still compile in release mode with this change?

Reply:
The `TRANSFER_DELAY_MILLIS` is indeed not really useful if the test fail anyway but at least it will be there if the `resolve_binary` issue is fixed.

Reply:
> But there _are_ more call sites to this, aren't there? In `linera-service`? Does `linera-service` still compile in release mode with this change?

The `ClientWrapper` and `LocalNetwork` where `resolve_binary` is used are only for tests, it is not used in `linera-service` binary

Reply:
and yes it compiles in release mode

Reply:
erratum: `LocalNetwork` is used in the `linera net` command, I will check if it works in release mode.

Reply:
and this command returns the same error as in the the tests (in the `main` branch).